docs/DeveloperGuide.md

```diff
@@ -1,38 +1,269 @@
-# Developer Guide
+<div align="center">
+<h2> MoneyMind Developer Guide</h2>
+<h3> "Mind your Money" </h3>
+</div>
+
+<div align="left">
 
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Design
```

Comment:
Seems like the markdown failed to render as expected from acknowledgements onwards. Consider checking your markdown before it might be potentially reviewed by anyone in the future.